Frameworks/UIKit/NSLayoutManager.mm

```diff
@@ -72,105 +85,107 @@ - (void)__layoutAllText {
         _frame.reset(CTFramesetterCreateFrame(framesetter.get(), {}, path.get(), nullptr));
     }
 
+    NSTextContainer* container = (NSTextContainer*)_textContainers[0];
+    const CGSize containerSize = container.size;
+    CGPoint origin{ 0, 0 };
+
     for (id lineRef in static_cast<NSArray*>(CTFrameGetLines(_frame.get()))) {
```

Comment:
it seems to be fine

---
In reply to: [92847146](https://github.com/Microsoft/WinObjC/pull/1558#discussion_r92847146) [](ancestors = 92847146)